src/olympia/abuse/management/commands/sync_cinder_policies.py

```diff
@@ -0,0 +1,15 @@
+from django.core.management.base import BaseCommand
+
+import olympia.core.logger
+from olympia.abuse.tasks import sync_cinder_policies
+
+
+class Command(BaseCommand):
+    help = (
+        'Sync all cinder policies to our local database. Syntax: \n'
+        '    ./manage.py sync_cinder_policies'
+    )
+    log = olympia.core.logger.getLogger('z.abuse')
```

Comment:
This is unused/pointless.

Reply:
Yep, artifact of copy paste. Removing.

Reply:
Actually ended up adding a log, otherwise the command prints nothing to the console which I find a bit unsettling. lmk what you think about that.

Reply:
Works for me. Maybe to be more accurate the log could say something like "Triggered policy sync task" or something like that but it doesn't really matter.

Reply:
Updated.